Tylko w gnome-terminal-2.16.1-new/autom4te.cache: output.1t
Tylko w gnome-terminal-2.16.1-new/autom4te.cache: traces.1t
diff -ur gnome-terminal-2.16.1/src/terminal-screen.c gnome-terminal-2.16.1-new/src/terminal-screen.c
--- gnome-terminal-2.16.1/src/terminal-screen.c	2006-10-02 21:08:52.000000000 +0200
+++ gnome-terminal-2.16.1-new/src/terminal-screen.c	2007-02-05 15:30:32.302872000 +0100
@@ -272,7 +272,7 @@
 #define USERCHARS "-A-Za-z0-9"
 #define PASSCHARS "-A-Za-z0-9,?;.:/!%$^*&~\"#'"
 #define HOSTCHARS "-A-Za-z0-9"
-#define PATHCHARS "-A-Za-z0-9_$.+!*(),;:@&=?/~#%"
+#define PATHCHARS "][A-Za-z0-9_$.+!*(),;:@&=?/~#%-"
 #define SCHEME    "(news:|telnet:|nntp:|file:/|https?:|ftps?:|webcal:)"
 #define USER      "[" USERCHARS "]+(:["PASSCHARS "]+)?"
 #define URLPATH   "/[" PATHCHARS "]*[^]'.}>) \t\r\n,\\\"]"
Tylko w gnome-terminal-2.16.1-new/src: terminal-screen.c~
